src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationService.kt

```diff
@@ -341,30 +341,31 @@ internal class RecommendationService(
     featureFlags: FeatureFlags?,
   ): RecommendationResponse {
     validateRecallType(jsonRequest)
-    val requestJson = JSONObject(jsonRequest.toString())
+    val requestJson = jacksonObjectMapper().readTree(jsonRequest.toString())
     val existingRecommendationEntity = getRecommendationEntityById(recommendationId)
 
-    val sendConsiderationRationaleToDelius = requestJson.optBoolean("sendConsiderationRationaleToDelius")
-    val considerationSensitive = requestJson.optBoolean("considerationSensitive")
+    val sendConsiderationRationaleToDelius = requestJson.has("sendConsiderationRationaleToDelius") &&
```

Comment:
You may be safe with just:
```kotlin
val sendConsiderationRationaleToDelius = requestJson.get("sendConsiderationRationaleToDelius").asBoolean()
```
Looks like `get` won't throw if the property doesn't exist and `asBoolean` will safely convert to `Boolean` defaulting to `false`

Reply:
It's true that `get` wont throw an exception on a null/non existent field, but it could return null, and so we couldn't call `asBoolean` on it without getting a null exception